packages/runtime/src/core.ts

```diff
@@ -92,6 +93,21 @@ export class FederationHost {
       ],
       void
     >('loadRemote'),
+    prefetch: new AsyncHook<
```

Comment:
@zhoushaw  Not sure if this name will cause confusion for users , because we have had preload hook

Reply:
Whats the difference betweent prefetch and preload?